analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -382,3 +387,6 @@ interface XmlPackageFileReader {
      */
     fun getDependencies(definitionFile: File): Set<NuGetDependency>
 }
+
+private fun String.isFixedVersion() =
+    listOf('(', ')', '[', ']', '*', '+').none { it in this } && isNotBlank()
```

Comment:
How about reusing `VERSION_RANGE_CHARS` (after adding `*` and `+` to it)? Also, maybe make the cheap `isNotBlank()` condition go first.

Reply:
Do you mean I should change the value of VERSION_RANGE_CHARS and thereby also the behavior of `Dependency.getId()` ?

Reply:
> Do you mean I should change the value of VERSION_RANGE_CHARS and thereby also the behavior of `Dependency.getId()` ?

Yes. The behavior change should not matter, but helps you to reuse the variable. This, however, should be stated in the commit message then.